examples/apps/nextjs-app/src/styles/globals.css

```
@@ -1,89 +1,92 @@
 @import 'tailwindcss';
 
+@source "./src/*.{ts,tsx}";
+@source "../../shared/design-system/src/**/*.{ts,tsx}";
+
+@import 'tw-animate-css';
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)